.github/workflows/on-workflow-call-blog-container-deployment.yml

```
@@ -1,15 +1,13 @@
 
 on:
   workflow_call:
 
 jobs:
-  push-container:
-    if: github.ref == 'refs/heads/main'
+  deploy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)